### PR TITLE
(PLATFORM-3357) Don't send dev CSP reports to prod service

### PR DESCRIPTION
**Pull request by Grunny**
/cc @Wikia/core-platform-team

### CODE REVIEW

File: includes/wikia/Wikia.php [hunk 1]
```diff
@@ -1252,8 +1252,14 @@ static public function onWebRequestInitialized( WebRequest $request ) {
 	 * @return bool true, it's a hook
 	 */
 	static public function outputHTTPSHeaders( WebRequest $request ) {
+		global $wgDevelEnvironment, $wgDevDomain;
 		if ( WebRequest::detectProtocol() === 'https' ) {
-			$request->response()->header( "Content-Security-Policy-Report-Only: default-src https:; script-src https: 'unsafe-inline' 'unsafe-eval'; style-src https: 'unsafe-inline'; img-src https: data:; report-uri " . self::CSP_ENDPOINT );
+			if ( !empty( $wgDevelEnvironment ) ) {
```
Comment (themech) [inline includes/wikia/Wikia.php]: Cannot we simply use `"services.{$wgWikiaBaseDomain}"` here?

Comment (Grunny) [reply includes/wikia/Wikia.php]: `wgWikiaBaseDomain` is always `wikia.com` even on devboxes. We could make a services domain config var that all code can use though, and move this to config. What do you think?

Comment (themech) [reply includes/wikia/Wikia.php]: Yes, I would prefer having services domain in config. Can be similar to how `wgDiscussionsApiUrl` is implemented.

Comment (Grunny) [reply includes/wikia/Wikia.php]: And done: https://github.com/Wikia/config/pull/2385

**Review by themech — APPROVED** [review]

**Pull request merged**